perf/terraform/region/main.tf

```diff
@@ -78,15 +78,15 @@ resource "aws_security_group" "restricted_inbound" {
   }
 
   ingress {
-    from_port   = 4001
-    to_port     = 4001
+    from_port   = 1
+    to_port     = 65535
```

Comment:
Why change the ingress port @MarcoPolo? For your zig server you only need egress, no?